specs/SUMMARY.md

```diff
@@ -45,7 +45,7 @@
   - [Configurability](./protocol/configurability.md)
 - [Experimental]()
   - [Custom Gas Token](./protocol/granite/custom-gas-token.md)
-  - [Plasma](./experimental/plasma.md)
+  - [Alt-DA](./experimental/alt-da.md)
```

Comment:
Should we add a redirect for this? Can be done in `book.toml`.

Reply:
And should we delete the old plasma.md? If it's not in `SUMMARY.md` it won't be rendered in the docs site anyway.

Reply:
Ya, my mistake, I pulled changes before pushing and didn't realize the file was restored.

Will add to redirect as well.

Reply:
redirect now added